joomla/joomla-cms
# FeaturedButton fix

# Summary of Changes

`FeaturedButton` has hard-coded task prefix `articles.` and uses COM_CONTENT-related tip strings.

It should work similarly to `PublishedButton`.

### Testing Instructions

Create your own FeaturedButton button with own `task_prefix` option.

Apply patch, ensure that feature/unfeature buttons work in backend articles.

### Actual result BEFORE applying this Pull Request

See that `articles.` still exist in button HTML.

### Expected result AFTER applying this Pull Request

Valid task name in custom button

Ensure that feature/unfeature buttons work in backend articles.

### Documentation Changes Required

No.
